pkgs/desktops/lxqt/core/lxqt-panel/default.nix

```diff
@@ -49,6 +49,13 @@ stdenv.mkDerivation rec {
     lxmenu-data
   ];
 
+  patches = [
+    (fetchurl {
```

Comment:
sorry I'm relatively new to nix and I don't really know whether it should be like this or whether I should use fetchPatch. I just took them from master, @romildo wrote the fix..